package.json

```diff
@@ -17,10 +22,23 @@
   },
   "devDependencies": {
     "autoprefixer": "^10.4.12",
-    "eslint": "8.24.0",
+    "eslint": "^8.0.1",
```

Comment:
Is there a reason for using a lower version of `eslint`?

Reply:
No. I have updated the version. Thanks